src/sdk/namespace_blob.js

```diff
@@ -12,7 +12,9 @@ const azure_storage = require('../util/new_azure_storage_wrap');
 const stream_utils = require('../util/stream_utils');
 const s3_utils = require('../endpoint/s3/s3_utils');
 const schema_utils = require('../util/schema_utils');
-const valid_attr_regex = /^[A-Za-z_][A-Za-z0-9_]+$/;
+const invalid_azure_md_regex = /[^a-zA-Z0-9_]/;
+const XATTR_RENAME_PREFIX = 'rename_';
+const XATTR_RENAME_KEY_PREFIX = 'XATTR_RENAME_PREFIX_';
```

Comment:
should be "rename_key_" according to 
// 'rename_key_123_invalid':'123-invalid'